presto-docs/src/main/sphinx/release/release-0.220.rst

```diff
@@ -0,0 +1,55 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+* Fix an issue where spatial joins executed after right outer joins would fail in certain cases. See :issue:`12672`.
+* Fix an issue where ``TRY`` cannot catch the failures in ``ROW`` comparisons with ``NULL`` fields and ``ARRAY`` comparisons with ``NULL`` elements.
+* Lazily build the hash table for map.
+
+Raptor Changes
+--------------
+* Integrate the optimized ORC writer into Raptor. Add configuration property ``storage.orc.optimized-writer-stage`` to enable and configure the optimized writer.
```

Comment:
A user will not know what "the optimized ORC writer" is.  Something like

`Improve performance of ORC write performance in Raptor.  Use configuration property `storage.orc.optimized-writer-stage`` to enable his performance enhancement.`